_data/payments.yml

```diff
@@ -141,6 +141,13 @@ websites:
       software: Yes
       doc: https://ravelin.zendesk.com/hc/en-us/articles/115001071665-Two-Factor-Authentication
 
+    - name: SixthContinent
+      url: https://sixthcontinent.com/
+      img: sixthcontinent.png
+      tfa: No
+      twitter: sixthcontinent
+      facebook: sixthcontinentcom
```

Comment:
Is it just me or does the Twitter and Facebook pages not exist? 🤔

Reply:
I think that those were the old ones, I just updated the twitter and Facebook handle with the new ones